### PR TITLE
Added Routing and Components

### PR DESCRIPTION
**Pull request by yogesh-**
- Added Basic Routing
- Broke down pages to small components

### CODE REVIEW

File: src/components/categories/categories.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+export const Categories = () => {
+  return (
+    <div class="shop-for">
+      <div class="shop-for-elements">
+        <h1 class="h1">🛍️SHOP FOR🛍️</h1>
```
Comment (chanchal-panpaliya) [inline src/components/categories/categories.jsx:5]: change - class into className

File: src/components/categories/categories.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <div class="shop-for-images flex-row">
+          <div class="img-container-one">
+            <img class="square-img-shop" src="/assets/men.jpg" alt="men" />
+            <a href="/html/product_listing.html">
```
Comment (chanchal-panpaliya) [inline src/components/categories/categories.jsx:9]: follow for img -> import img_name from '../../path'; then use src={img_name}

File: src/components/categories/categories.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            <img class="square-img-shop" src="/assets/women.jpg" alt="women" />
+            <a href="/html/product_listing.html">
+              <button class="btn">
+                <p class="h4">WOMEN's</p>
+              </button>
+            </a>
+          </div>
+          <div class="img-container-three">
+            <img class="square-img-shop" src="/assets/kids.jpg" alt="kids" />
+            <a href="/html/product_listing.html">
```
Comment (chanchal-panpaliya) [inline src/components/categories/categories.jsx:25]: use link tag - react-router-dom

File: src/components/footer/footer.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+// import react from "react";
+
+export const Footer = () => {
+  return (
+    <div class="footer-home flex-col">
+      <h1 class="h3">Created by Yogesh Gupta</h1>
+      <div class="social" id="social">
+        <a href="https://github.com/yogesh-">
```
Comment (chanchal-panpaliya) [inline src/components/footer/footer.jsx:8]: you can also return this data by using - constant file . make array of object eg.  let data = [  {img:"../path", link:"http://" , name:"facebook" } , {.....} ,{.....}]    fetch -> data.map((item,index)=>{ return ( <li> {item.img}  <li> ) }) ... so on

File: src/pages/cart/cart.jsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { Footer } from "../../components/footer/footer";
+import { Navbar } from "../../components/navbar/navbar";
+
```
Comment (agrawalshanu708) [inline src/pages/cart/cart.jsx:3]: try to make index.js in seperate folder, and import from that

File: src/pages/home/home.jsx [hunk 6]
```diff
@@ -1,3 +1,8 @@
+import { Categories } from "../../components/categories/categories";
+import { Footer } from "../../components/footer/footer";
+import { Hero } from "../../components/hero/hero";
+import { Navbar } from "../../components/navbar/navbar";
+import { Trending } from "../../components/trending/trending";
 import "./home.css";
```
Comment (agrawalshanu708) [inline src/pages/home/home.jsx:7]: same here

**Review by chanchal-panpaliya — APPROVED** [review]
code looks good , need to - refactor code .

File: src/routing.jsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+export const Routing = () => {
+  return (
+    <div>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/products" element={<Products />} />
+        <Route path="/cart" element={<Cart />} />
+        <Route path="/wishlist" element={<Wishlist />} />
+        {/* <Route path="/signin" element={<SignIn />} />
+        <Route path="/signup" element={<SignUp />} /> */}
```
Comment (agrawalshanu708) [inline src/routing.jsx:16]: try to make seperate folder of routing

**Review by agrawalshanu708 — COMMENTED** [review]
else is good to merge

File: src/routing.jsx (hunk 7 above)
Comment (tushargoyal812) [inline src/routing.jsx:16]: remove comments.

**Review by tushargoyal812 — COMMENTED** [review]
1+ for what chanchal suggested,these are some mandatory changes you need to do.
files structure is good.
overall naming convention is also good.

File: src/components/categories/categories.jsx (hunk 1 above)
Comment (yogesh-) [reply src/components/categories/categories.jsx]: done

File: src/components/categories/categories.jsx (hunk 3 above)
Comment (yogesh-) [reply src/components/categories/categories.jsx:33]: done

File: src/components/footer/footer.jsx (hunk 4 above)
Comment (yogesh-) [reply src/components/footer/footer.jsx:8]: okay,

File: src/pages/cart/cart.jsx (hunk 5 above)
Comment (yogesh-) [reply src/pages/cart/cart.jsx:3]: ok

File: src/routing.jsx (hunk 7 above)
Comment (yogesh-) [reply src/routing.jsx:16]: okay

**Pull request merged**